src/celestia/celestiacore.cpp

```diff
@@ -4555,8 +4552,7 @@ bool CelestiaCore::isRecording()
 
 void CelestiaCore::flash(const string& s, double duration)
 {
-    if (hudDetail > 0)
```

Comment:
i changed this to ignore HudDetail value because i feel that it does not fit with other HUDs, this is more like a message shown to the user.

Reply:
`hudDetail == 0` means "don't show any information on the screen", so it's better to restore it